app/assets/stylesheets/wpcc/section/_contributions.scss

```diff
@@ -23,6 +19,9 @@
   padding: $baseline-unit*2;
   margin-bottom: $baseline-unit*2;
   background-color: $color-grey-concrete;
+    .form__row {
+      margin-bottom: 0;
+    }
```

Comment:
is this block correctly indented? It seems like should have a tab less if I read it correctly.

Reply:
good spot, thx